Makefile

```diff
@@ -210,10 +214,18 @@ ifneq ($(USE_GPERFTOOLS), 1)
 				endif
 			endif
 		endif
+		JEMALLOC_CUSTOM=$(shell pkg-config --exists jemalloc; echo $$?)
```

Comment:
Please use := so that it is only executed once

Reply:
Does this work on all operating systems that use Makefile, i.e. RHEL, CENTOS, ubuntu, MAC, etc?